### PR TITLE
Bumped the Nvidia chart version to support GPU metrics.

### PR DESCRIPTION
**Pull request by Gilbert88**
This PR bumps the Nvidia chart version to 0.2.0, which contains the GPU metrics and granfana dashboard support.

It also removes the limits for GPU driver daemonset, because GPU is considered as dedicated resources and we do not want to add limited resources to those mission critical tasks like GPU driver, to avoid dependent applications being affected if the driver being OOM killed.

This PR depends on the helm chart PR https://github.com/mesosphere/charts/pull/285

Screen Shot 2019-11-26 at 3 27 32 PM
Screen Shot 2019-11-26 at 3 29 20 PM

### CODE REVIEW

File: addons/nvidia/nvidia.yaml [hunk 1]
```diff
@@ -54,4 +51,16 @@ spec:
         initContainers:
         - name: init-wait
           image: busybox
-          command: ['sh', '-c', 'sleep 90']
+          command: ['sh', '-c', 'sleep 180']
```
Comment (joejulian) [inline addons/nvidia/nvidia.yaml]: Same comment as https://github.com/mesosphere/kubeaddons-configs/pull/350#discussion_r355059770

File: addons/nvidia/nvidia.yaml [hunk 2]
```diff
@@ -54,4 +51,16 @@ spec:
...
+      nvidia-dcgm-exporter:
+        enabled: true
+        namespace: kubeaddons
+        nodeSelector:
+          konvoy.mesosphere.com/gpu-provider: NVIDIA
+        initContainers:
+        - name: init-wait
+          image: busybox
+          command: ['sh', '-c', 'sleep 200']
```
Comment (joejulian) [inline addons/nvidia/nvidia.yaml]: here, too, about sleep

File: addons/nvidia/nvidia.yaml [hunk 3]
```diff
@@ -26,19 +26,16 @@ spec:
...
     repo: https://mesosphere.github.io/charts/staging
-    version: 0.1.2
+    version: 0.2.0
     values: |
       ---
       nvidia-driver:
         image:
           tag: "418.87.01-centos7"
         resources:
-          limits:
```
Comment (joejulian) [inline addons/nvidia/nvidia.yaml]: same as it's clone PR:
@hectorj2f You've done a ton of work on resource parameters. Do you have any issue with this change?

File: addons/nvidia/nvidia.yaml (hunk 1 above)
Comment (Gilbert88) [reply addons/nvidia/nvidia.yaml]: replied on that thread:)

File: addons/nvidia/nvidia.yaml (hunk 2 above)
Comment (Gilbert88) [reply addons/nvidia/nvidia.yaml]: ditto

File: addons/nvidia/nvidia.yaml (hunk 3 above)
Comment (Gilbert88) [reply addons/nvidia/nvidia.yaml]: see the other thread

**Comment by Gilbert88** [comment]
> This addon is currently not tested as part of our CI as it is [intentionally disabled for testing](https://github.com/mesosphere/kubernetes-base-addons/blob/master/test/groups.yaml#L64).
> 
> However, would this not be a great opportunity to remove it and get it tested? @Gilbert88 is it possible to test at least deploying the addon on a [kind](https://github.com/kubernetes-sigs/kind) cluster?

It will run on Soak. Do we want to set up GPU machines for CI? It will be a bit expensive

**Comment by joejulian** [comment]
Right, no, he cannot test this in a kind cluster. That would require the Teamcity runners to have GPU and those are really expensive.

**Comment by Gilbert88** [comment]
Just rebased to address conflicts

**Pull request merged**